docs/upgrading.rst

```diff
@@ -1,5 +1,25 @@
 Upgrading
 =========
+Docker-compose
+--------------
+When you deploy a vanilla docker-compose, it will create a persistent volume for your MySQL database. As long as your volume is there, you should not lose any data.
+
+The generic upgrade method for docker-compose follows these steps:
+- Download the latest or tagged images from https://hub.docker.com/repository/docker/defectdojo/defectdojo-django and https://hub.docker.com/repository/docker/defectdojo/defectdojo-nginx
+- Go to the directory where your docker-compose.yml file lives
+- Stop DefectDojo: ``docker-compose stop``
+- Re-start DefectDojo, allowing for container recreation: ``docker-compose up -d``
+- Run the database migrations to bring your database schema up to speed with the latest code  
+- If you have the initializer disabled (or if you want to be on the safe side), run the migration command: ``docker-compose exec uwsgi /bin/bash -c 'python manage.py migrate``
+
+If you build your images locally and do not use the ones from DockerHub, the instructions are much the same, except that you'd build your images first.
+Replace the first step above with this one:
+- ``docker-compose build``
```

Comment:
During the migration  (from v. 1.7.1 to v 1.10.1) I tried to rebuild the images, but the uwsgi failed. How can I fix this conflict?

Step 15/27 : RUN pip3 install   --no-cache-dir  --no-index   --find-links=/tmp/wheels   -r ./requirements.txt
 ---> Running in b158c6e5e38b
Looking in links: /tmp/wheels
Collecting django-custom-field
  Cloning https://github.com/DefectDojo/django-custom-field.git (to revision 4e095dd9119858e90fd21560e5618c6347e4ef1b) to /tmp/pip-install-riu1eckl/django-custom-field_9bcb4627696a42c4ad62d9eb34322f72
Collecting django-tastypie-swagger
  Cloning https://github.com/DefectDojo/django-tastypie-swagger.git (to revision c7c5629480dc20cadd3015490a01bd8e70393c36) to /tmp/pip-install-riu1eckl/django-tastypie-swagger_6779dca6e23e4078b48d9c5942f934da
Processing /tmp/wheels/asteval-0.9.21-py3-none-any.whl
Processing /tmp/wheels/bleach-3.2.1-py2.py3-none-any.whl
Processing /tmp/wheels/celery-4.4.7-py2.py3-none-any.whl
Processing /tmp/wheels/coverage-5.3-cp36-cp36m-manylinux1_x86_64.whl
Processing /tmp/wheels/cpe-1.2.1-py3-none-any.whl
Processing /tmp/wheels/cryptography-3.2.1-cp35-abi3-manylinux2010_x86_64.whl
Processing /tmp/wheels/defusedxml-0.6.0-py2.py3-none-any.whl
Processing /tmp/wheels/Django-2.2.17-py3-none-any.whl
Processing /tmp/wheels/django_auditlog-0.4.7-py3-none-any.whl
INFO: pip is looking at multiple versions of django to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of defusedxml to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of cryptography to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of cpe to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of coverage to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of celery to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of bleach to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of asteval to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install -r ./requirements.txt (line 8) and python-dateutil==2.8.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested python-dateutil==2.8.1
    django-auditlog 0.4.7 depends on python-dateutil==2.6.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
ERROR: Service 'uwsgi' failed to build: The command '/bin/sh -c pip3 install    --no-cache-dir  --no-index   --find-links=/tmp/wheels   -r ./requirements.txt' returned a non-zero code: 1